app/controlplane/pkg/biz/workflowcontract.go

```diff
@@ -45,10 +45,11 @@ type WorkflowContract struct {
 }
 
 type WorkflowContractVersion struct {
-	ID        uuid.UUID
-	Revision  int
-	CreatedAt *time.Time
-	Schema    *Contract
+	ID           uuid.UUID
```

Comment:
I belive there are two objects no? A contract and a version, why haven't we instead put in in the workflow?

It feels wrong that we are adding a property from another object into the version.

Reply:
ok, I see now, it's because of the biz/repo calls that we make.

What if we just update the `FindVersionByID` to return `WorkflowContractWithVersion` instead of `WorkflowContractVersion`?

That will make it clear what we are returning and this change will affect only that function.